internal/rpcserver/interface.go

```diff
@@ -456,3 +456,15 @@ type LogManager interface {
 	// is invalid.
 	ParseAndSetDebugLevels(debugLevel string) error
 }
+
+// SanityChecker represents a block sanity checker for use with the RPC server.
+// The purpose of this is to allow an alternative implementation to be used for
+// testing.
```

Comment:
While it's true that this allows for an alternative implementation for testing, this is also a good enhancement outside of testing since it simplifies what rpcserver needs to know about checking block sanity.

Reply:
noted, will remove the second line.